### DISCUSSION TITLE
Basic Constraints

### DISCUSSION THREAD

**Discussion by zquestz** [FiloSottile/mkcert#583] [Ideas]
Is there any way to also include "Basic Constraints" on this cert? Perhaps with a flag?

The latest ASUS firmware requires it for uploading a custom cert, and this tool did a great job before they added that requirement.

I was able to patch mkcert to get it working for my router, but not sure these changes are ideal. For those curious -> https://github.com/zquestz/mkcert/commit/8e6f1214971366d17ddb80804456894bc779904f

**Comment by dan-alan** [comment]
Thanks! Much needed
